src/eu/eutampieri/catacombs/model/Gun.java

```diff
@@ -0,0 +1,24 @@
+package eu.eutampieri.catacombs.model;
+
+public class Gun extends SimpleWeapon{
```

Comment:
Ah tu dici di fare così? In realtà ci sta anche

Reply:
si cosi dopo per altre armi che potremmo mettere si estende solo simple e si vola